scripts/sync_repo_files.sh

```diff
@@ -53,28 +53,41 @@ post_pull_request() {
     "https://api.github.com/repos/${1}/pulls"
 }
 
+check_license() {
+  # Check to see if the input is an Apache license of some kind
+  echo "$1" | grep --quiet --no-messages --ignore-case 'Apache License'
```

Comment:
```suggestion
  grep --quiet --no-messages --ignore-case 'Apache License' "$1"
```

Reply:
The input is a string with the contents of the target file, not a filename.

Reply:
oh OK